src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -46,6 +47,9 @@
     public static final String ADDRESS_DESC_BOB = " " + PREFIX_ADDRESS + VALID_ADDRESS_BOB;
     public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
+    public static final String DELTAG_DESC_FRIEND = " " + PREFIX_DELTAG + VALID_TAG_FRIEND;
+    public static final String DELTAG_DESC_HUSBAND = " " + PREFIX_DELTAG + VALID_TAG_HUSBAND;
+    public static final String DELTAG_ALL = " " + PREFIX_DELTAG + "all";
```

Comment:
Would be better to make `all` a `public static final String`

Reply:
It seems to be used very specificaly so i thought there wasnt a need to redirect one more time, but ill consider it